content/cover_letter.md

```diff
@@ -0,0 +1,32 @@
+Scott Markel  
+Dassault Systèmes BIOVIA  
+5005 Wateridge Vista Dr  
+San Diego, CA 92121
+
+October 30th, 2020
+
+Dear Dr. Markel,
+
+We are writing to submit our manuscript entitled “Ten Quick Tip for Deep Learning in Biology” for consideration in _PLOS Computational Biology_'s education section.
+
+Deep learning provides new opportunities for classifying and recognizing patterns in biological data.
```

Comment:
I would change "classifying and recognizing patterns in biological data" to "for outcome prediction and pattern recognition."